src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -520,8 +518,6 @@ protected virtual Task WriteJsonResponseAsync<TResult>(HttpContext context, Http
         return _serializer.WriteAsync(context.Response.Body, result, context.RequestAborted);
     }
 
-    /****** WebSocket support *********
```

Comment:
Search all `await next` lines across the repo. I see 3 usages without `ConfigureAwait(false)`.

Reply:
There is not a single use of ConfigureAwait(false) in this project. Probably because it generally isn’t necessary on ASP.NET Core. I thought we would add that to all code but I want to finish all my merges first.

Reply:
There will probably be 100s of references that need to be fixed.

Reply:
> There is not a single use of ConfigureAwait(false) in this project.

![изображение](https://user-images.githubusercontent.com/21261007/170372129-7b2adf23-1a25-47b2-b59c-49b6a56224c8.png)

Reply:
Not many then.

Reply:
lol - that line was one you added

Reply:
For consistency, I should remove it...